students/nartowska_karina/lesson_11_argparse_and_math/regex_search.py

```diff
@@ -0,0 +1,35 @@
+import re
+import glob
+import argparse
+import sys
+
+
+def check_arg(args=None):
+    parser = argparse.ArgumentParser(description='Regex search')
+    parser.add_argument('-r', '--regex',
+                        help='search regex',
+                        required='True',
+                        default='')
```

Comment:
The ```default``` argument is not needed in the case when you defined the ```required``` flag. Please briefly look at:
https://docs.python.org/3.6/library/argparse.html#default
https://docs.python.org/3.6/library/argparse.html#required